update/update-version.md

```diff
@@ -0,0 +1,129 @@
+        - Copy the **VPC ID** from the Overview tab.
+    1. Get VPC ID from Projects:
+        - In the {{site.data.keyword.cloud_notm}} console, click the **Navigation menu** icon ![Navigation menu icon](../icons/icon_hamburger.svg "Menu"), and then click **Projects**.
+        - Select the Project associated with your VSI deployment.
+        - Click on the **Configurations** tab.
+        - Select the configuration associated with your VSI deployment.
+        - In the **Outputs** tab you can find an output labeled **vpc_data**.
+        - Clicking on it will open a json of all the VPC data of the VPCs provisioned.
+        - Copy the VPC IDs from the **vpc_data** json.
+
+1. [Optional] If the VSIs are being deployed in an account that is different from the account where the Schematics Workspace are located, you need to provide an IBM Cloud API key, which can be used for fetching the VPC details. In this case, you can pass the VPC API-KEY to the script using `-k` flag. For more information, see https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui
+
+1. Run the script from any local machine
+```sh
+bash schematics_update_v3.x.x_to_v4.x.x.sh -v "<vpc-id1>,[<vpc-id2>,...]" -r "<vpc-region> [-k <vpc-ibm-api-key>]"
+```
+This script will trigger a new job in the schematics workspace, please monitor the state of the job in IBM Cloud UI. Check if the job completes **Successfully**, in that case go ahead with **Step 5**. In case of schematics workspace job ends in a failed state, please follow **Step 7** to revert any changes made by the script.
+
+1. Now pull in the latest release in Schematics and click on `Generate plan` and make sure none of the VSIs will be recreated.
+
+1. Click on `Apply plan`.
+
+1. [Optional] If a schematics workspace job that was initiated through the script encounters an issue, you can undo any modifications made by running the script again with the -z flag. For example,
+```sh
+bash schematics_update_v3.x.x_to_v4.x.x.sh -z
+```
+A new schematics workspace job reverting the state back to its prior condition, which existed prior to the execution of the script.
+
+## Local Terraform State file
+{: #local}
+
+If you have both the code and the Terraform state file stored locally on your machine, you can utilize the update_v3.x.x_to_v4.x.x.sh script to avoid the recreation of Virtual Server Instances (VSIs).
+
+1. Make sure you have all the dependencies to run the script.
+    - IBM Cloud CLI
+    - IBM Cloud CLI 'is' plugin
+    - Terraform CLI
+    - jq
+
+1. Set the `IBMCLOUD_API_KEY` variable as an environment variable.
+    1. IBMCLOUD_API_KEY : This is an IBM Cloud API key, which can be used for accessing the VPC. For more information, see https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui
+    ```sh
+    export IBMCLOUD_API_KEY="<API-KEY>" #pragma: allowlist secret
+    ```
+
+1. Retrive the VPC ID of the VPC on which all the VSIs are deployed, including the region of VPC.
```

Comment:
we never mentioned how to get the region. I don't actually see a region value exposed in the output. I do see it in some CRNs though, should we parse it?. Or should add a region to the output (would have come in a PR that was merged before this release though). Or can we just assume people will know what region their VPC is in?